packages/request-tracker/lib/xhr-tracker.js

```diff
@@ -63,6 +77,7 @@ function createXhrTracker (global, options = {}) {
           onRequestEnd({
             endTime: Date.now(),
             state: 'error',
+            headers: this.getAllResponseHeaders(),
             xhr: this
```

Comment:
It looks like we're passing the `xhr` object back just to get the response body in `plugin-http-errors`. I think it would be better to have a `body` property on the `endContext` and set that here instead of passing the `xhr` instance to `onRequestEnd`. The `startContext` and `endContext` should be a consistent interface for both xhr and fetch.